packages/lit-dev-tools-cjs/src/search/indexers/index-external-data.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import * as fs from 'fs/promises';
+import * as path from 'path';
+import type {UserFacingPageData} from '../plugin';
+
+export async function indexExternalData(
+  outputDir: '_dev' | '_site',
+  idOffset = 0
+) {
+  if (outputDir === '_dev') {
+    return [];
+  }
+
+  // Path of the video index.
+  const EXTERNAL_DATA_INDEX_PATH = path.resolve(
+    __dirname,
+    // Load the article content itself not the tags pages.
```

Comment:
Update these comments please. Both should refer to external data, instead of video and article.